backfire/ema.py

```diff
@@ -27,10 +27,10 @@ def set_lower_window(self, val):
         self.lower_window = val
     factor_high = 0
     def set_factor_high(self, val):
-        self.factor_high = val
+        self.factor_high = 1.0 + val
```

Comment:
This will return 1.xxx and 0.9xxx in the results. I think we should handle it in the logic file instead.